lua/starfall/instance.lua

```diff
@@ -509,6 +509,15 @@ local function cpuRatio(instance)
 	return instance:movingCPUAverage() / instance.cpuQuota
 end
 
+SF.Instance.Ram = 0
+SF.Instance.RamAvg = 0
+local function ramRatio()
+	local ram = collectgarbage("count")
+	SF.Instance.Ram = ram
+	SF.Instance.RamAvg = SF.Instance.RamAvg*0.999 + ram*0.001
+	return ram / SF.RamCap:GetInt()
```

Comment:
if this is called often it might be better to localize this and update it with a convar callback, could probably done in more places too